### PR TITLE
Fix links to docs

### PR DESCRIPTION
**Pull request by tcmetzger**
Since updating the structure of our docs, some URLs have changed which broke some of the links in the tutorial. This PR aims to update links to the new structure!

### CODE REVIEW

**Comment by tcmetzger** [comment]
I'm also using this PR to fix some text aroud the vbar example in chapter 4 - seems like some text from the line chart example used to be copied there, so I replaced this with some text that actually matches the example!

File: notebooks/07_annotations.ipynb [hunk 1]
```diff
@@ -624,9 +624,9 @@
  ],
  "metadata": {
   "kernelspec": {
-   "display_name": "scipy-scipy-interactive-dataviz-bokeh",
+   "display_name": "bk-tutorial",
    "language": "python",
-   "name": "conda-env-scipy-scipy-interactive-dataviz-bokeh-py"
+   "name": "python3"
```
Comment (bryevdv) [inline notebooks/07_annotations.ipynb]: Not sure about this change, @pavithraes ?

Comment (tcmetzger) [reply notebooks/07_annotations.ipynb]: This was committed accidentally!

Comment (tcmetzger) [reply notebooks/07_annotations.ipynb]: I fixed chapter 7 and 8, they should now use the correct metadata again. I'm working on this from a train with a somewhat unpredictable internet connection... 😅

**Review by bryevdv — APPROVED** [review]

**Review by pavithraes — APPROVED** [review]
Thank you!

**Pull request merged**